_drafts/2017-09-01-first-review-experiences.md

```diff
@@ -0,0 +1,137 @@
+---
+name: first-review-experiences
+layout: post_discourse
+title: Experiences as a first time rOpenSci package reviewer
+date: 2017-09-01
+authors:
+  - name: Verena Haunschmid
+    url: http://blog.haunschmid.name
+tags:
+- R
+- software
+- review
+- ropensci
+- onboarding
+---
+
+It all started January 26^th^ this year when I signed up to volunteer as
+a reviewer for R packages submitted to rOpenSci. My main motivation for
+wanting to do package reviews was to learn something new and to
+contribute to the R open source community.
+
+Three months later I was contacted by [Ma�lle Salmon](https://twitter.com/ma\_salmon) asking whether I was interested in
+reviewing the R package ['patentsview'](https://github.com/ropensci/patentsview) for rOpenSci. And yes, I
+was! To be honest I was a little bit thrilled.
+
+The packages are submitted for review to rOpenSci via an issue to their
+github repository and also the reviews happen there. So you can check out all
+previous package submissions and reviews. With all the information you
+get from rOpenSci and also the help from the editor it is straight
+forward to do the package review. Before I started my review I read the
+reviewer guides (links below) and checked out a few of the existing
+reviews. I installed the package 'patentsview' from github and also
+downloaded the source code so I could check out how it was implemented.
+There are several R packages that helped me get started with my review,
+e.g. ['devtools'](https://github.com/hadley/devtools) and
```

Comment:
Replace ['devtools'] with [`devtools`] and ['goodpractice'] with [`goodpractice`] and check all other package names to use backticks instead of single quote symbols. This will format package names correctly in the post.